x-pack/plugins/cases/README.md

```diff
@@ -52,14 +52,39 @@ Arguments:
 |disabledStatuses?|`CaseStatuses[];` array of disabled statuses
 |getCaseDetailsHref|`(caseDetails: CaseDetailsHrefSchema) => string;` callback to generate the case details url from the case id
 |isModal?|`boolean;` is All Cases table a modal
-|onCaseDetailsNavClick|`(caseDetails: CaseDetailsHrefSchema) => void;` callback for nav click
-|onConfigureCasesNavClick?|`(ev: React.MouseEvent) => void;` callback for nav click
-|onCreateCaseNavClick?|`(ev: React.MouseEvent) => void;` callback for nav click
+|onCaseDetailsNavClick|`(caseDetails: CaseDetailsHrefSchema) => void;` callback for case details nav click
+|onConfigureCasesNavClick?|`(ev: React.MouseEvent) => void;` callback for configure case nav click
+|onCreateCaseNavClick?|`(ev: React.MouseEvent) => void;` callback for create case nav click
 |onRowClick?|`(theCase?: Case ! SubCase) => void;` callback for row click, passing case in row
-|userCanCrud|boolean; user permissions to crud
+|userCanCrud|`boolean;` user permissions to crud
 
 UI component:
  ![All Cases Component][all-cases-img] 
+ 
+### `getCaseView`
+Arguments: 
+
+|Property|Description|
+|---|---|
+|allCasesHref|`string;` route for all cases page
```

Comment:
Thanks for adding these :) I'm gonna try and clean up this API once everything is deleted to make it easier to consume now that we know what most of them are (apart from the timeline ones left)